x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/translations.ts

```diff
@@ -26,7 +26,7 @@ export const ACTIVITY_LOG = {
       unisolatedAction: i18n.translate(
         'xpack.securitySolution.endpointDetails.activityLog.logEntry.action.unisolated',
```

Comment:
i don't know if this is necessary but should we also change the id as well?

Reply:
maybe. I left it only because in the code we are still using `unisolate` through api's etc.